test/factories/github/pull_request_reviews.rb

```diff
@@ -0,0 +1,20 @@
+FactoryBot.define do
+  factory :github_pull_request_review, class: 'Github::PullRequestReview' do
+    pull_request { create :github_pull_request }
+    node_id { "MDE3OlB1bGxSZXF1ZXN0UmV2aWV3NTk5ODA2NTI4" }
+    reviewer_username { "ErikSchierboom" }
+
+    trait :random do
+      pull_request { create :github_pull_request, :random }
+      node_id { SecureRandom.hex }
+    end
+
+    before(:create) do |r|
+      r.pull_request.update!(
+        data: r.pull_request.data.tap do |d|
+          d[:reviews] = [{ node_id: r.node_id, reviewer_username: r.reviewer_username }]
+        end
+      )
```

Comment:
Is there any way to make this a bit prettier? Basically, the parent (Git::PullRequest) needs to have data with values that are defined in this factory.